nservicebus/hosting/ibus-instance.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Managing IBus instance
+summary: Fluent configuration API to get transnational one-way messaging
+tags:
+- Hosting
+- IOC
+- Dependency Injection
+- Service Locator
+---
+
+
+When you have created a `IBus` instances then you must register it somewhere so that your application can use it to send messages.
+
+There are various ways in managing and storing the `IBus` instance.
+
+* Global static
+* Using dependency injection
+
+The first option creates tight coupling the last loose coupling.
+
+
+
+## Global static
+
+After you have created the bus you assign the object to a global static variable so that the bus can be used in other locations
+
+
+Initializing:
+
+```
+public class Global
+{
+    public static IBus Bus { get; set; }
+}
+
+Global.Bus = NServiceBus.Create(busConfiguration).Start();
+```
+
+Using it:
+
+```
+Global.Bus.Send(new MyCommand());
+```
+
+Cleanup:
+
+```
+Global.Bus.Dispose();
+```
+
+This is very simple to use but creates a lot of tight coupling which makes unit testig pretty hard.
+
+## Dependency injection container
+
+Here is the description of the dependency injection pattern:
+
+>In software engineering, dependency injection is a software design pattern that implements inversion of control for software libraries. Caller delegates to an external framework the control flow of discovering and importing a service or software module specified or "injected" by the caller.
+
+Source: https://nl.wikipedia.org/wiki/Dependency_injection
```

Comment:
* replace:  This is very simple...
* with: We do not recommend using the static IBus definition, as it introduces tight coupling, and creates difficulties when unit testing. 

* remove the Dependency inection container header and its section. This is not needed. Note the Dutch wikipedia link instead of the English one...